api/src/main/java/com/mtech/vmcs/service/impl/initialfiledata/InitialFileDataBean.java

```diff
@@ -0,0 +1,48 @@
+package com.mtech.vmcs.service.impl.initialfiledata;
+
+import com.mtech.vmcs.model.entity.Coin;
+import com.mtech.vmcs.model.entity.Drink;
+import com.mtech.vmcs.model.entity.Machine;
+import com.mtech.vmcs.model.entity.User;
+
+import java.util.List;
+
+public class InitialFileDataBean {
```

Comment:
The class name implies it's a ```Spring bean```, while actually it's just a data model. So I reckon ```InitialFileData``` may be better, and put it into  ```model/entity``` dir.

Reply:
Using ``` @Setter @Getter ``` annotations can simplify the code.